glossary.yml

```diff
@@ -5521,6 +5521,14 @@
       a single program in parallel. Orginally designed for use
       in graphics, but is also used for general computation in
       the form of [compute shaders][#compute_shader].
+  uk:
```

Comment:
@Herobread checked rendered version at https://glosario.carpentries.org/uk/#gpu - both English and Ukrainian have wrong hyperlink formatting - the 2nd pair of brackets must be `(...)` 

Would you like to submit a PR to fix both?